scapy/contrib/isotp.py

```diff
@@ -31,7 +31,6 @@
 from scapy.compat import chb, orb
 from scapy.layers.can import CAN, CAN_MAX_IDENTIFIER, CAN_MTU, CAN_MAX_DLEN
 import scapy.modules.six as six
-import scapy.automaton as automaton
```

Comment:
I will manually add this changes to #3188, since you probably want this PR to be in, first.

Reply:
Thanks a lot !